README.md

```diff
@@ -1,20 +1,25 @@
 # Mattermost Custom Attributes Plugin [![CircleCI](https://circleci.com/gh/mattermost/mattermost-plugin-custom-attributes.svg?style=svg)](https://circleci.com/gh/mattermost/mattermost-plugin-custom-attributes)
 
-This plugin allows you to add custom attributes to users in your Mattermost. Currently it only exposes those attributes in the profile popover of users but in the future the plan is to extend the plugin to allow adding badges next to usernames based on attributes and potentially display the attributes in other places.
+This plugin adds custom attributes to users in your Mattermost instance.
 
-We use this plugin on https://community.mattermost.com to distinguish Mattermost core committers and staff from other users.
+Currently the plugin only exposes the specified attributes in the user profile popover, but this plugin can be extended to allow displaying attributes elsewhere in the user interface, such as badges next to usernames.
 
-**Supported Mattermost Server Versions: 5.4+**
```

Comment:
we're not including the server version info anymore?

Reply:
I was thinking of noting it in the Releases page https://github.com/mattermost/mattermost-plugin-custom-attributes/issues/4

This because
1. 5.4 was released almost a year ago
2. Anyone who upgrades to the version this plugin ships pre-packaged won't find this note relevant

That said, I'm only 1/5 on it.

Reply:
@jasonblais seems fine to me then!